config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
`qemu-system-x86_64` is the system (distro) installed default, no? So, until all of our q35 updates are upstream and mainlined into all common distros, I don't think this will be good - unless you were thinking specifically to place that binary in a virtcontainers or CC specific area. I'd still prefer it still didn't have the same name as the system default though.